### ISSUE TITLE
Documentation menu not scrollable on mobile.

### ISSUE THREAD

**Issue by slanelb** [plentico/plenti.co#10]
This occurs on mobile Safari. Don't know about other browsers.
![image](https://user-images.githubusercontent.com/66718039/112639722-7f4d0980-8e38-11eb-84b8-747ca4f933e2.png)

**Comment by jimafisk** [comment]
Good catch @slanelb, I was experiencing this as well. I made some updates to the mobile menu, is it any better on your end now? Thanks!

**Comment by slanelb** [comment]
The documentation Nav is now overflowing its box so it's visible but not acting as you would want it with a scroll. Here's a screenshot.
![image](https://user-images.githubusercontent.com/66718039/113475859-a7a7ba00-946f-11eb-8bcf-70e23c8cbba0.png)
